src/Pos/Lrc/Consumers.hs

```diff
@@ -14,6 +18,8 @@ import           Pos.Ssc.Class.Workers (SscWorkersClass (sscLrcConsumers))
 import           Pos.Update.Lrc        (usLrcConsumer)
 
 allLrcConsumers
-    :: (LrcMode ssc m, SscWorkersClass ssc)
+    :: forall ssc m.
+       (LrcMode ssc m, SscWorkersClass ssc)
     => [LrcConsumer m]
-allLrcConsumers = [delegationLrcConsumer, usLrcConsumer] ++ untag sscLrcConsumers
+allLrcConsumers = [delegationLrcConsumer, usLrcConsumer] ++
+                  untag (sscLrcConsumers @ssc)
```

Comment:
It's strange to see usage of `@ssc` and `untag` at the same time. AFAIK, they should solve same problem.